lighthouse-cli/test/smokehouse/test-definitions/oopif-scripts.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/** @type {LH.Config.Json} */
+const config = {
+  extends: 'lighthouse:default',
+  categories: {
+    performance: {
+      title: 'Performance',
+      auditRefs: [{id: 'oopif-iframe-test-audit', weight: 0}],
+    },
+  },
+  audits: [
+    // Include an audit that *forces* the IFrameElements artifact to be used for our test.
+    {path: 'oopif-iframe-test-audit'},
+  ],
+  settings: {
+    // This test runs in CI and hits the outside network of a live site.
+    // Be a little more forgiving on how long it takes all network requests of several nested iframes
+    // to complete.
+    maxWaitForLoad: 180000,
+  },
+  passes: [
+    // CI machines are pretty weak which lead to many more long tasks than normal.
+    // Reduce our requirement for CPU quiet.
+    {
+      passName: 'defaultPass',
+      cpuQuietThresholdMs: 500,
+    },
+  ],
+};
+
+/**
+ * @type {Smokehouse.ExpectedRunnerResult}
+ * Expected Lighthouse audit values for sites with OOPIFS.
+ */
+const expectations = {
+  lhr: {
+    requestedUrl: 'http://localhost:10200/oopif-scripts.html',
+    finalUrl: 'http://localhost:10200/oopif-scripts.html',
+    audits: {
+      'network-requests': {
+        details: {
+          items: {
+            _includes: [
+              {url: 'http://localhost:10200/oopif-scripts.html'},
+              {url: 'http://localhost:10200/oopif-simple-page.html'},
+              {url: 'http://localhost:10503/oopif-simple-page.html'},
+              {url: 'http://localhost:10200/simple-script.js'},
+              {url: 'http://localhost:10503/simple-script.js'},
+              {url: 'http://localhost:10200/simple-script.js'},
+              {url: 'http://localhost:10503/simple-script.js'},
+              {url: 'http://localhost:10200/simple-worker.js'},
+              {url: 'http://localhost:10503/simple-worker.js'},
```

Comment:
for some reason simple-worker.mjs never showed up....

Reply:
that's fine! I just want to document things as they are now.

I added a new `_excludes` assertion so we will know when this stops happening with future refactors

Reply:
:pikaface:

I just tried with ToT chrome and .mjs showed up...